os/arch/arm/src/amebasmart/amebasmart_spi.c

```diff
@@ -1383,9 +1386,9 @@ FAR struct spi_dev_s *amebasmart_spibus_initialize(int bus)
 	
 	if (bus == 0) {
 		/* Select SPI0 */
-
+#ifdef CONFIG_AMEBASMART_SPI0
 		priv = &g_spi0dev;
```

Comment:
This change will cause hardfault in the later line 1395 or 1397.
Maybe it should return null here?